Test Runner/Test Runner.xcodeproj/project.pbxproj

```diff
@@ -77,6 +79,8 @@
 		7552A7472A573CBB0023DA5A = {
 			isa = PBXGroup;
 			children = (
+				8AE5AE902B85498900FE3759 /* UI Tests.xctestplan */,
+				8AE5AE8F2B85495400FE3759 /* Unit Tests.xctestplan */,
```

Comment:
These can go under a `Test Plans` folder

Reply:
Hmm not sure what to change as those two files are in a Test Plans folder in the top level directory.

Are these just references?